src/Driver/Pdo/AbstractPdoCommand.php

```diff
@@ -265,14 +265,18 @@ protected function logQuery(string $rawSql, string $category): void
 
     protected function queryInternal(int $queryMode): mixed
     {
-        $rawSql = $this->getRawSql();
         $logCategory = self::class . '::' . $this->getQueryMode($queryMode);
 
-        $this->logQuery($rawSql, $logCategory);
+        $this->logger?->log(LogLevel::INFO, $rawSql = $this->getRawSql(), [$logCategory]);
```

Comment:
If `logQuery()` not used in adapters, we can mark his deprecated and remove before next major release.